app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,62 @@
+class PasswordResetsController < ApplicationController
+  before_action :load_user, :valid_user, :check_expiration,
+                only: %i(edit update)
+
+  def new; end
+
+  def edit; end
+
+  def create
+    @user = User.find_by email: params[:password_reset][:email].downcase
```

Comment:
`params[:password_reset][:email]` mà nil thì downcase sống ko cu
tìm hiểu về `dig` đi em, và toán tử an toàn &. hoặc try nữa hì